### PR TITLE
add option to $cloneEmbed

### PR DESCRIPTION
**Pull request by Discord-Verified-Developer**
## Description

Added an option to return the json format for the cloned embed.

## Type of change

- [ ] Bug fix (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

---

- [x] My code follows the style guidelines of this project
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/functions/cloneEmbed.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ module.exports = async (d) => {
         d.embeds.push(embeds[index - 1]);
     }
 
-    d.embeds = d.embeds.filter((embed) => embed);
+    if (!returnJSON) d.embeds = d.embeds.filter((embed) => embed);
+    else data.result = d.embeds
```
Comment (Faf4a) [inline src/functions/cloneEmbed.js:26]:
```suggestion
    d.embeds = d.embeds.filter((embed) => embed);
    if (returnJSON) data.result = d.embeds
```

**Review by Faf4a — COMMENTED** [review]
Thanks for the pull request, I left some changes!

**Comment by Discord-Verified-Developer** [comment]
I merged your suggested changes

**Review by Faf4a — COMMENTED** [review]
.

**Review by Faf4a — APPROVED** [review]
LGTM

**Pull request merged**